Dockerfile-dev

```diff
@@ -0,0 +1,19 @@
+FROM python:3.8
+
+RUN apt-get -y update && \
```

Comment:
if `wait-for-it` not used in an entrypoint script, this RUN-command can be removed.

Reply:
you still need the update and upgrade, no?

Reply:
Bo, the image provides all needed (up-to-date) software. The update is necessary to fetch the apt-repositories, so apt can find wait-for-it. I think upgrade wasn't necessary at all. But maybe you ask Gernot for a confirmation on this.

Reply:
@gsiv ?

Reply:
Gernot and I removed the commando and changed to image back to alpine